src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -140,4 +140,12 @@ public boolean equals(Object other) {
                 || (other instanceof UniquePersonList // instanceof handles nulls
                         && this.internalList.equals(((UniquePersonList) other).internalList));
     }
+
+    /**
+     * Sorts all persons in the list according to their full name
+     */
+    public void sort(){
```

Comment:
coding style violation. [read this](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html)